packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -696,6 +807,10 @@ abstract class Element implements BuildContext {
   Widget get widget => _widget;
   Widget _widget;
 
+  BuildOwner _owner;
```

Comment:
trivial nit: we normally put the property below the getter to avoid the dartdoc being sandwiched between the two.